userguide/tutorials/notification_plugin.adoc

```diff
@@ -15,37 +15,45 @@ The https://github.com/killbill/killbill-hello-world-java-plugin[Hello World Plu
 Let us take a closer look at the `HelloWorldListener` class:
 
 [source,java,linenums]
-public class HelloWorldListener implements OSGIKillbillEventDispatcher.OSGIKillbillEventHandler{
-    private static final Logger logger = LoggerFactory.getLogger(HelloWorldListener.class);
-    private final OSGIKillbillAPI osgiKillbillAPI;
-    public HelloWorldListener(final OSGIKillbillAPI killbillAPI) {
-        this.osgiKillbillAPI = killbillAPI;
-    }
-    @Override
-    public void handleKillbillEvent(final ExtBusEvent killbillEvent) {
-        logger.info("Received event {} for object id {} of type {}",
-                    killbillEvent.getEventType(),
-                    killbillEvent.getObjectId(),
-                    killbillEvent.getObjectType());
-        final TenantContext context = new PluginTenantContext(killbillEvent.getAccountId(), killbillEvent.getTenantId());
-        switch (killbillEvent.getEventType()) {
-            //
-            // Handle ACCOUNT_CREATION and ACCOUNT_CHANGE only for demo purpose and just print the account
-            //
-            case ACCOUNT_CREATION:
-            case ACCOUNT_CHANGE:
-                try {
-                    final Account account = osgiKillbillAPI.getAccountUserApi().getAccountById(killbillEvent.getAccountId(), context);
-                    logger.info("Account information: " + account);
-                } catch (final AccountApiException e) {
-                    logger.warn("Unable to find account", e);
-                }
-                break;
-            // Nothing
-            default:
-                break;
+public class HelloWorldListener implements OSGIKillbillEventDispatcher
+    .OSGIKillbillEventHandler{
```

Comment:
You cannot re-order the code like this, otherwise the text won't match the line numbers anymore on Windows.

If you were unable to fix the CSS for the line numbering, that's ok -- just revert as it was before and we can find another solution for the future.